specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -451,10 +651,32 @@
             "$ref": "#/definitions/PrivateLinkServiceConnection"
           },
           "description": "A grouping of information about the connection to the remote resource. Used when the network admin does not have access to approve connections to the remote resource."
+        },
+        "customDnsConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomDnsConfig"
+          },
+          "description": "A grouping of information about dns configurations connecting to the remote resource."
         }
       },
       "description": "Properties of the private endpoint."
     },
+    "CustomDnsConfig":{
+      "properties": {
+        "fqdn": {
+          "type": "string",
+          "description": "The fqdn for the remote resource."
```

Comment:
should it be constrained to these?https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-overview#dns-configuration

Reply:
Sorry I'm not sure what this means, are you maybe saying the fqdn has to be constrained to the host, privzone, custom dns zone options? I'm not sure what the would mean ultimately for the code.